lib/go-qmstr/database/dgraph.go

```diff
@@ -167,30 +167,33 @@ func (db *DataBase) queueWorker() {
 	}
 }
 
-func (db *DataBase) checkFileData(node *service.FileNode) error {
+func (db *DataBase) checkFileData(node *service.FileNode) (string, error) {
```

Comment:
But  now this is not a check anymore. Rename the function to reflect its purpose.